pkg/resources/grant_privileges_to_account_role_acceptance_test.go

```diff
@@ -1065,3 +1110,28 @@ func queriedAccountRolePrivilegesContainAtLeast(roleName sdk.AccountObjectIdenti
 		})
 	}, roleName, privileges...)
 }
+
+func createExternalVolumeTemporarily(t *testing.T, externalVolumeName string) {
```

Comment:
Why the `temporarily`?

Reply:
removed `temporarily` and returned cleanup, so it's consistent with `testint` package utils